bedrock/mozorg/templates/mozorg/manifesto-base.html

```diff
@@ -28,13 +28,13 @@
   <div class="side-reference">
     <h4 class="side-reference-title">{{ _('Our Products') }}</h4>
     <p>{{ _('Software and other innovations designed to advance our mission.') }}</p>
-    <a class="more" href="{{ url('firefox') }}">{{ _('Learn More') }}</a>
```

Comment:
I noticed that this is technically a different string than some other ones since "More" is capitalized. Hopefully it's fine to change that, but I did want to note the change.

Reply:
Yeah - we currently have two different strings in `main.lang`, one with `more` and another with `More`. Rather than keeping two, I decided to opt for the most prevalent occurence which is the former. I think this likely makes more sense than migrating two different occurences, as many locales seem to ignore the capitalization. Maybe this is something we could have a preference with the the `en` file only, which we can always change later on?

Reply:
could probably change it with CSS for en-* locales too. But it does bring up the question of whether there are locales that translated the two differently, and which they'd prefer in the migration. Maybe we need some L10n team input here?

Reply:
Localized strings from the main.lang, and only a handful are different. 
- [Learn more](https://transvision.mozfr.org/string/?entity=mozilla_org/main.lang:d59048f21fd887ad520398ce677be586&repo=mozilla_org).
- [Learn More](https://transvision.mozfr.org/string/?entity=mozilla_org/main.lang:f5395c9793af8a11b406ca7c1ac70da9&repo=mozilla_org).

Let's keep the one that agrees with the current style.

Reply:
Peiying says there's not much difference between the two string in terms of translations, so I think we're ok to keep one and not the other. Good call asking to verify 👍